README.md

```diff
@@ -20,13 +20,30 @@ Project DeepSpeech is an open source Speech-To-Text engine that uses a model tra
 * [TensorFlow 1.0 or above](https://www.tensorflow.org/install/)
 * [SciPy](https://scipy.org/install.html)
 * [PyXDG](https://pypi.python.org/pypi/pyxdg)
-* [python_speech_features](https://pypi.python.org/pypi/python_speech_features)
 * [python sox](https://pypi.python.org/pypi/sox)
 * [pandas](https://pypi.python.org/pypi/pandas)
+* [DeepSpeech native client libraries](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/)
 
 
 ## Install
 
+### Installing pre-built DeepSpeech Python bindings
+
+Pre-built binaries can be found on [TaskCluster](https://tools.taskcluster.net/index/artifacts/#project.deepspeech.deepspeech.native_client.master/project.deepspeech.deepspeech.native_client.master.cpu). You'll need to download `native_client.xz` and the appropriate Python wheel package.
```

Comment:
A Debian package is a nice idea, though I think follow-up (I don't run Debian so it's not easy for me to do it quickly :)) - And I'll link to both in that case, I guess it isn't going to hurt.

And that's right, all MacOS users would have to go the build-from-source path right now... I'm not that happy about that, but I guess it's a choice between keeping around deprecated code or that for now :/ Unless we figure out Mac cross-compilation I suppose?